ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/data/PaywallState.kt

```diff
@@ -265,6 +265,12 @@ internal sealed interface PaywallState {
                 if (currentTabContainsThisPackage) selectedPackageByTab[currentTabIndex] = selectedPackage
             }
 
+            fun getDefaultSelectedPackage(): Package? {
+                return initialSelectedPackageOutsideTabs
+                    ?: selectedPackageByTab[selectedTabIndex]
+                    ?: packages.packagesByTab[selectedTabIndex]?.firstOrNull()?.pkg
```

Comment:
I don't have a lot of context, but should this logic be inverted? Like:

```suggestion
                return packages.packagesByTab[selectedTabIndex]?.firstOrNull()?.pkg
                    ?: selectedPackageByTab[selectedTabIndex]
                    ?: initialSelectedPackageOutsideTabs
```

Just thinking that we should try to keep the previously selected package within the tab after dismissing the sheet? Not sure if this makes a lot of sense though, I think I'm missing context on the issue.

Reply:
> selectedPackageByTab​ is empty and packagesByTab​ as well

I do think that's normal if the paywall doesn't have tabs. Those are only used in that case. 

> So we have decided to just select the default package back when closing the sheet.

Ok, we should probably test what happens in case we have tabs with different packages on each tab (to make sure we return to the default tab + default package within that tab. But I think in general the code makes sense!

Reply:
made a little change and the logic is now:
- First select default pacakge in tab
- If null select default package outside of tabs